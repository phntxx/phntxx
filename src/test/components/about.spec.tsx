import { render } from "@testing-library/react";
import About from "../../components/about";

describe("about.tsx", () => {
  it("should render component without crashing", () => {
    const { asFragment } = render(<About />);
    expect(asFragment).toMatchSnapshot();
  });
});
