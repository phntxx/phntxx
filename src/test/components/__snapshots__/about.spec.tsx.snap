// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`about.tsx should render component without crashing 1`] = `[Function]`;
